onprc_ehr/resources/views/snomed.html

```diff
@@ -0,0 +1,176 @@
+<style>
+    .hiddenEl {
+        display: none;
+    }
+
+    .errorMsg {
+        color: #FF7377;
+    }
+</style>
+
+<script type="text/javascript" nonce="<%=scriptNonce%>">
+
+    const searchBySubset = false;
+
+    function onEdit() {
+        document.getElementById("createSnomed").style.display = "none";
+        document.getElementById("postSearch").innerHTML = "";
+        document.getElementById("results").innerHTML = "";
+        document.getElementById("showMessage").innerHTML = "";
+    }
+
+    function showMessage(msg, error) {
+        document.getElementById("showMessage").innerHTML = msg;
+        if (error) {
+            document.getElementById("showMessage").className = "errorMsg";
+        }
+        else {
+            document.getElementById("showMessage").className = "";
+        }
+    }
+
+    function createSnomed() {
+
+        const subset = document.getElementById("subsetOptions").value;
+        const snomed = document.getElementById("snomed").value;
+
+        LABKEY.Ajax.request({
+            url: LABKEY.ActionURL.buildURL("onprc_ehr", "saveSnomed"),
+            method: 'POST',
+            jsonData: {
+                subset: 'Organisms', // TODO: change to subset if supporting subset dropdown
```

Comment:
TODO still relevant?

Reply:
Just for my curiosity, why is the subset "Organisms"?

Reply:
Right now we're just scoping this to Organisms. I left a few hints in here how to make this work for more subsets, which could be something we'll be doing later.